new_ui/templates/partials/compose.html

```
@@ -71,59 +71,88 @@
             </div>
         </div>
         <div class="quick-settings__item">
             <label class="text-label">Seed</label>
             <div style="display: flex; align-items: center; gap: var(--space-2);">
-                <button class="btn btn-secondary btn-sm" disabled title="Random seed">🎲</button>
-                <input type="text" class="input input-sm text-mono" value="-1" disabled
+                <button type="button" class="btn btn-sm"
+                        :class="randomSeed ? 'btn-primary' : 'btn-secondary'"
+                        @click="toggleRandomSeed()"
+                        title="Random seed">🎲</button>
+                <input type="number" class="input input-sm text-mono"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)